addon/templates/components/frost-textarea.hbs

```diff
@@ -12,7 +13,14 @@
   tabindex=tabindex
 }}
 {{#if showClear}}
-  {{#frost-button class='clear' type='icon' tabIndex='-1' onClick=(action 'clear')}}
-    {{frost-icon  icon='close'}}
+  {{#frost-button
```

Comment:
Should just replace this whole thing with a div/svg target like we have in `frost-text`